packages/thirdweb/src/chainsaw/endpoints/getBlock.ts

```diff
@@ -0,0 +1,41 @@
+import type { GetBlockReturnType } from "viem";
```

Comment:
Does chainsaw use viem for this?

Reply:
Ideally we minimize viem dependency and just port their code over (where the types are manageable). This will also break our doc generation if we directly return a viem type

Reply:
Ported their type over [a341578](https://github.com/thirdweb-dev/js/pull/3741/commits/a3415782b19bcc9ad322de9e4e5ef3dc7ff242a1)